docroot/themes/humsci/humsci_basic/src/scss/elements/_base.scss

```diff
@@ -68,3 +68,26 @@ h6:not([class]) {
 p:not([class]) {
   @include hb-body--small;
```

Comment:
I'm feeling tempted to change this base element style to `@include hb-body--medium'`, (which is 18px) because it's the base font size on the node detail page. Any concerns there? Would that impact any of your work?

Reply:
sounds fine to me!